src/bitwise/core.cljs

```diff
@@ -47,14 +47,16 @@
                           :grid-template-rows "30px"
                           :align-items "center"})
 
-(def action-button-styles {:width 100
+(def action-button-styles {:min-width 100
                            :height 25
                            :border "1px solid gray"
-                           :display "flex"
+                           :display "inline-block"
```

Comment:
Does this still center the button content?

Reply:
Originally I had swapped out for inline-block to get the button layout behavior I was looking for. But I was able to get the same thing by adding "flex" to the parent. 👍